### PR TITLE
Translated the Rendering part

### PR DESCRIPTION
**Pull request by EugeneYak**

### CODE REVIEW

**Comment by EugeneYak** [comment]
Ooh my night! 
By the way, how would you translate the word Rendering into Russian? Рендеринг? Отдача?

File: README-ruRU.md [hunk 1]
```diff
@@ -237,22 +237,21 @@
...
 
-<!--- @FIXME: Translate this snippet. --->
-### Rendering
+### Рендеринг
 
 * <a name="inline-rendering"></a>
-  Prefer using a template over inline rendering.
-  <sup>[[link](#inline-rendering)]</sup>
+  Отдавайте предпочтение шаблонам вместо отдачи линейного кода.
+  <sup>[[ссылка](#inline-rendering)]</sup>
```
Comment (arbox) [inline README-ruRU.md]: Линейный код? Please try to find a more appropriate translation.

File: README-ruRU.md [hunk 2]
```diff
@@ -269,30 +268,29 @@
...
 * <a name="plain-text-rendering"></a>
-  Prefer `render plain:` over `render text:`.
-  <sup>[[link](#plain-text-rendering)]</sup>
+  Используйте `render plain:` вместо `render text:`.
+  <sup>[[ссылка](#plain-text-rendering)]</sup>
 
   ```Ruby
-  # bad - sets MIME type to `text/html`
+  # плохо - задается MIME-тип `text/html`
   ...
```
Comment (arbox) [inline README-ruRU.md]: Please use parens for comments as used throughout this guide: "Statement (some comments)". Consistency is very important.

File: README-ruRU.md [hunk 3]
```diff
@@ -269,30 +268,29 @@
...
   render plain: 'Ruby!'
   ...
   ```
-<!--- @FIXME --->
 * <a name="http-status-code-symbols"></a>
-  Prefer [corresponding symbols](https://gist.github.com/mlanett/a31c340b132ddefa9cca) to numeric HTTP status codes.
-  They are meaningful and do not look like "magic" numbers for less known HTTP status codes.
-  <sup>[[link](#http-status-code-symbols)]</sup>
+  Используйте [символы, соответствующие](https://gist.github.com/mlanett/a31c340b132ddefa9cca) числовым кодам состояния HTTP.
```
Comment (arbox) [inline README-ruRU.md]: Only one word as link in brackets.

**Pull request closed without merging**